packages/flutter_client_sdk/README.md

```diff
@@ -11,6 +11,12 @@
 
 [![Twitter Follow](https://img.shields.io/twitter/follow/launchdarkly.svg?style=social&label=Follow&maxAge=2592000)](https://twitter.com/intent/follow?screen_name=launchdarkly)
 
+## Platform Support
```

Comment:
This seems to be the way most packages represent this, so I added it here. The caveats about compatibility are in the top level README.md. Should be sufficient.